execution_workflows/Roar/workflow/config/config.Roar.yaml

```diff
@@ -0,0 +1,19 @@
+# Configuration file for snakemake pipeline
```

Comment:
This file seems to be duplicated (you mention to delete test data, just want to make sure this is also tagged).